### ISSUE TITLE
guihua.lua which is not present in plugins  table

### ISSUE THREAD

**Issue by Migo1989** [ray-x/guihua.lua#16]
![image](https://user-images.githubusercontent.com/7768408/186406681-aa9cf4c1-9f46-4f59-b5f1-4662146d156a.png)

ray-x/guihua.lua has already downloaded
![image](https://user-images.githubusercontent.com/7768408/186406911-89138c86-c4b5-4dbd-b508-d061573730c8.png)

![image](https://user-images.githubusercontent.com/7768408/186406962-ce3f74f0-6732-4009-887a-5330c097770c.png)

**Comment by ray-x** [comment]
how about run `packadd guihua.lua` directly?

**Comment by Migo1989** [comment]
thanks

**Comment by wwnbb** [comment]
Got the same issue with navigator.
<img width="1320" alt="Screenshot 2023-03-21 at 16 50 32" src="https://user-images.githubusercontent.com/5105340/226664629-f713ee9a-284d-421b-9214-33df263a019f.png">

**Comment by craigrosie** [comment]
Is there any update on this issue? I'm getting the same issue: `Error: attempted to load guihua.lua which is not present in plugins table!`

I tried the playground that you linked to. I don't get the `guihua.lua` error, but none of the `navigator.lua` keymaps work, so I'm not sure if I've done something wrong.

Steps I took:
* `git clone git@github.com:ray-x/navigator.lua.git`
* `cd navigator.lua/playground/`
* `nvim -u init.lua`
* ☝️ installs all the neovim packages. `treesitter` installation failed, but it didn't tell me why
* exit neovim
* nvim -u init.lua py/`
* open `hello.py` and move my cursor over the `Dog` class
* try keymaps like `gK`, `<esc>gr`, etc, but nothing happens
